wallet/src/main/java/com/android/identity_credential/wallet/credman/CredmanPresentationActivity.kt

```diff
@@ -22,32 +22,17 @@ import android.util.Base64
 import androidx.fragment.app.FragmentActivity
 import androidx.lifecycle.lifecycleScope
 import com.android.identity.android.mdoc.util.CredmanUtil
-import com.android.identity.android.securearea.AndroidKeystoreKeyUnlockData
-import com.android.identity.android.securearea.UserAuthenticationType
-import com.android.identity.cbor.Cbor
-import com.android.identity.credential.Credential
 import com.android.identity.mdoc.credential.MdocCredential
 import com.android.identity.document.DocumentRequest
-import com.android.identity.document.NameSpacedData
 import com.android.identity.crypto.Algorithm
 import com.android.identity.crypto.Crypto
 import com.android.identity.crypto.EcCurve
-import com.android.identity.crypto.EcPublicKey
 import com.android.identity.crypto.EcPublicKeyDoubleCoordinate
-import com.android.identity.issuance.DocumentExtensions.documentConfiguration
-import com.android.identity.mdoc.mso.MobileSecurityObjectParser
-import com.android.identity.mdoc.mso.StaticAuthDataParser
 import com.android.identity.mdoc.response.DeviceResponseGenerator
-import com.android.identity.mdoc.response.DocumentGenerator
-import com.android.identity.mdoc.util.MdocUtil
-import com.android.identity.securearea.KeyLockedException
-import com.android.identity.securearea.KeyUnlockData
 import com.android.identity.util.Constants
 import com.android.identity.util.Logger
-import com.android.identity_credential.wallet.R
 import com.android.identity_credential.wallet.WalletApplication
-import com.android.identity_credential.wallet.presentation.showPresentmentFlow
-import com.android.identity_credential.wallet.ui.prompt.biometric.showBiometricPrompt
+import com.android.identity_credential.wallet.presentation.showMdocPresentmentFlow
```

Comment:
We really only want a single `showPresentmentFlow()` method so I think `showMdocPresentmentFlow()` needs to go, right?

Reply:
The complication is that we need slightly different parameters between mdoc and sd-jwt, when they need to sign and generate the response.

I've refactored this a bit so there's only a single function, but this means there are 3 arguments that the caller needs to be careful with:
- encodedSessionTranscript is only needed for mdoc
- nonce and clientId are only needed for sd-jwt.

That pushes these to runtime NPD errors, rather than compile-time argument errors. IMHO, that's more problematic than having two functions. What do you think?

Reply:
OK, just leave it with two functions and we'll refactor this later...

Reply:
Alright, back to two functions (more clearly named this time).